tempesta_fw/t/functional/helpers/wrk.py

```diff
@@ -0,0 +1,57 @@
+""" Wrk script generator """
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2017 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+import os
+
+class ScriptGenerator(object):
+    """ Generate lua script """
+    request_type = "GET"
+    uri = "/"
+    headers = []
+    body = ""
+    filename = None
+    def __luaencode(self, value):
```

Comment:
Method can be a simple function. But why it's actually required? It does just nothing.

Reply:
It should care about escaping, if we write test with "\"" and so on in content

Reply:
Now we don't have tests with content, which should be escaped. So now it does nothing